tests/e2e/execute-tests-cloud-function.sh

```diff
@@ -17,7 +17,7 @@ START_LOAD_GENERATION=$(date -u +%s%3N)
 export START_LOAD_GENERATION
 
 ./tests/e2e/deployment-test-cloud-function.sh
-sleep 420
+sleep 600
```

Comment:
Those `sleep` statements are fine but, in the future, we should be aiming for some kind of verification/check procedures if script execution results are in effect (i.e. call GCP API if cloud function is deployed).

Reply:
Yes, right. For now, I was just trying different values, but I need to push them to test. If any change could be made, I will discuss it first.